addon/PageWorker.js

```diff
@@ -0,0 +1,39 @@
+const {Page} = require("sdk/page-worker");
+const {data} = require("sdk/self");
+const {LOCAL_STORAGE_KEY} = require("common/constants");
+const {ADDON_TO_CONTENT} = require("common/event-constants");
+
+class PageWorker {
+  constructor({store}) {
+    this._page = null;
+    this._onDispatch = null;
+    this._unsubscribe = null;
+    this._store = store;
```

Comment:
Since it's expected to be called with a `store` property should we throw an error if `store` is `undefined`?